### PR TITLE
docs(typography): update the typography docs

### PR DESCRIPTION
**Pull request by simeonoff**

### CODE REVIEW

File: en/components/themes/typography.md [hunk 1]
```diff
@@ -6,149 +6,210 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
...
+There are a several mixins and functions that are used to set and retrieve category styles to/from a type scale. Those are:
+
+- `igx-type-style` [function] - Returns a set of style rules to be used by a type scale category.
+- `igx-type-scale` [function] - Returns a set of 13 style categories.
+- `igx-type-scale-category` [function] - Returns a map of style rules from a type scale and category.
+- `igx-type-style` [mixin] - Adds style rules to a selector from a specific type scale and category.
+- `igx-typography` [mixin] - Defines the global application typography styles.
+
+
+Let's take a close look at what each one of the aforementioned mixins and functions do.
```
Comment (ig-ealbert) [inline en/components/themes/typography.md]: Let's take a closer look?

File: en/components/themes/typography.md [hunk 2]
```diff
@@ -6,149 +6,210 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
...
+);
+```
+
+> [!NOTE]
+> Any properties that you do not pass, such as `$font-family`, `letter-spacing`, etc. will be automatically replaced with the default values as specified in the `$default-type-scale` for the category you want to use your style for.
+
+
+#### The Type Scale
+
+The type scale is a map of type styles for all 13 scale categories. To generate a new type map all you have to do is write.
```
Comment (ig-ealbert) [inline en/components/themes/typography.md]: To generate a new type map, all you have to do is write the following SCSS:
"SCSS" here is changeable.  I don't know what the best word would be.  "rule"?  "style"?

File: en/components/themes/typography.md [hunk 3]
```diff
@@ -6,149 +6,210 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
...
+Typography style mixins can be scope to specific selectors. Say we wanted our custom card typography styles to be applied for all `igx-card` components with class name of `my-cool-card`. 
+
+```scss
+//...
+.my-cool-card {
+    @include igx-card-typography($type-scale: $my-type-scale);
+}
+```
+
+The typography component mixins take a second argument - `$categories`. It is used to configure which parts of the component use what typography scale category. For instance, if we wanted the our custom card to use a different scale category for the title than `h5`, we could change it.
```
Comment (ig-ealbert) [inline en/components/themes/typography.md]: ... if we wanted [the/our - choose one!] custom card...
Ex. "if we wanted our custom card"...

File: en/components/themes/typography.md [hunk 4]
```diff
@@ -6,149 +6,210 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
...
+        title: 'h6'
+    )
+);
+```
+
+Now the card component will use the `overline` scale category to style the title. The user can completely overhaul the entire card typography by assigning different type scales to the different text parts of the card.
+
+### CSS Classes
+
+In addition to adding text styles for all components based on type scale categories, we also style the default h1-h6 and p elements. We also separate semantics from styling. So fo instance, even though the `h1` tag has some default styling that we provide when using `igx-typography`, you can modify it to look like an `h3` by giving it a class of `igx-typography__h3`.
```
Comment (ig-ealbert) [inline en/components/themes/typography.md]: So for instance

File: jp/components/themes/typography.md [hunk 5]
```diff
@@ -2,153 +2,214 @@
 title: Typography
 _description:
 _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library
-_language: ja
```
Comment (ig-ealbert) [inline jp/components/themes/typography.md]: Please do not remove this!

File: jp/components/themes/typography.md [hunk 6]
```diff
@@ -2,153 +2,214 @@
...
-        </blockquote>
-        <div class="spec">
-            <span>フォント サイズ: <b>1.5em</b> (24px)</span>
-            <span>行の高さ: <b>2.185em</b> (35px)</span>
-            <span>上のマージン: <b>0</b></span>
-            <span>下のマージン: <b>1.6875rem</b> (27px)</span>
-        </div>
-    </div>
-</article>
+Let's take a close look at what each one of the aforementioned mixins and functions do.
```
Comment (StefanIvanov) [inline jp/components/themes/typography.md]: closer

File: jp/components/themes/typography.md [hunk 7]
```diff
@@ -2,153 +2,214 @@
...
+@include igx-typography(
+    $font-family: "'Roboto', sans-serif",
+    $type-scale: $my-type-scale,
+    $base-color: #484848
+);
+```
+
+### Custom Type Styles
+The `igx-type-style` mixin can be used to retrieve the style rules for a scale category from a specific type scale. Further, it allows you to add additional style rules.
+
```
[the remark this answers is not in the document]
Comment (ig-ealbert) [reply jp/components/themes/typography.md]: Yes, that is ok.  I think either one is ok.

File: jp/components/themes/typography.md [hunk 8]
```diff
@@ -6,14 +6,14 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
...
+<p class="highlight">Ignite UI for Angular Typography Sass モジュールは、アプリケーション全体、特定ののタイポグラフィのスケールやコンポーネントを変更できます。</p>
 <div class="divider"></div>
 
-Ignite UI for Angular follows [The Type System](https://material.io/design/typography/the-type-system.html#) as described in the Material Design specification. The type system is a ***type scale*** consisting of ***13 different category type styles*** used across most components. All of the scale categories are completely reusable and adjustable by the end user.
+Ignite UI for Angular は、マテリアル デザインの仕様で説明されるように[タイプ システム](https://material.io/design/typography/the-type-system.html#)に準拠します。タイプ システムは、***タイプ スケール***でほとんどのコンポーネントで使用されている **13 の異なるカテゴリ タイプ スタイル**で構成されます。エンドユーザーはすべてのスケール カテゴリの完全な再利用、調整ができます。
 
-Here's a list of all 13 category styles as defined in Ignite UI for Angular:
-| **Scale Category** | **Font Family** | **Font Weight** | **Font Size** | **Text Transform** | **Letter Spacing** | **Line Height** |
+以下は、Ignite UI for Angular で定義される 13 カテゴリ スタイルすべてのリストです。
+| **スケール カテゴリ** | **フォント ファミリ** | **フォント ウェイト** | **フォント サイズ** | **テキスト変換** | **文字スペース** | **行の高さt** |
```
Comment (ig-ealbert) [inline jp/components/themes/typography.md]: @yoshimis The "t" should be removed at the end of the last column header.

**Comment by zdrawku** [comment]
Typography is merged to master and will be available with the planned 6.2.0-beta.0 version

**Pull request merged**